<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="elihle" failures="1" tests="3" name="com.mystore.testcases.RegisterAccountLandingPageTest" time="185.911" errors="0" timestamp="2025-07-24T16:18:02 SAST" skipped="2">
  <testcase classname="com.mystore.testcases.RegisterAccountLandingPageTest" name="registrationAccountTest" time="101.189">
    <skipped/>
  </testcase> <!-- registrationAccountTest -->
  <system-out/>
  <testcase classname="com.mystore.testcases.RegisterAccountLandingPageTest" name="registrationAccountTest" time="35.657">
    <skipped/>
  </testcase> <!-- registrationAccountTest -->
  <system-out/>
  <testcase classname="com.mystore.testcases.RegisterAccountLandingPageTest" name="registrationAccountTest" time="49.065">
    <failure message="expected [Register Account1] but found [Register Account]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Register Account1] but found [Register Account]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.mystore.testcases.RegisterAccountLandingPageTest.registrationAccountTest(RegisterAccountLandingPageTest.java:50)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- registrationAccountTest -->
  <system-out/>
</testsuite> <!-- com.mystore.testcases.RegisterAccountLandingPageTest -->
